sql/core/src/test/resources/sql-tests/results/postgreSQL/strings.sql.out

```diff
@@ -977,7 +977,7 @@ struct<repeat(Pg, -4):string>
 -- !query
 SELECT trim(binary('\\000') from binary('\\000Tom\\000'))
```

Comment:
so we already support binary by casting binary to string?

Reply:
The current implementation only supports `string`, but because `ImplicitCastInputTypes` is used, `binary` is cast into `string`.